cogni-libs/src/main/java/zone/cogni/libs/sparqlservice/impl/HttpClientUtils.java

```diff
@@ -0,0 +1,89 @@
+package zone.cogni.libs.sparqlservice.impl;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.Authenticator;
+import java.net.PasswordAuthentication;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Utility methods for JDK11 HttpClient.
+ */
+public class HttpClientUtils {
+
+  private static final Logger log = LoggerFactory.getLogger(HttpClientUtils.class);
+
+  /**
+   * Creates a basic authenticator (username, password)
+   *
+   * @param username username
+   * @param password password (not null)
+   * @return authenticator
+   */
+  public static Authenticator createBasicAuthenticator(final String username,
```

Comment:
is this basic as in "simple" or as in basic-auth

in first case, the name is confusing
in 2nd case, not sure this is related

Reply:
Looks like only Basic authentication is implemented in JDK client https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/376ab432aa0d120ace1077bc71bc694b1af86e0a/src/java.net.http/share/classes/jdk/internal/net/http/AuthenticationFilter.java#L276.

Anyway I removed the method as it is only used once and is not needed for now.